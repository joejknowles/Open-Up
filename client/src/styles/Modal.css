.modal {
    position: fixed; /* Stay in place */
    z-index: 1; /* Sit on top */
    left: 0;
    top: 0;
    width: 100%; /* Full width */
    height: 100%; /* Full height */
    overflow: auto; /* Enable scroll if needed */
    background-color: rgb(0,0,0); /* Fallback color */
    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
}

.modal-box {
  min-height: 100vh;
  background-color: #fefefe;
  margin: 5% auto; /* 15% from the top and centered */
  padding: 20px;
  border: 1px solid #888;
  width: 60%; /* Could be more or less, depending on screen size */
  -webkit-box-shadow: 6px 6px 100px -8px rgba(0,0,0,0.75);
  -moz-box-shadow: 6px 6px 100px -8px rgba(0,0,0,0.75);
  box-shadow: 6px 6px 100px -8px rgba(0,0,0,0.75);
}
