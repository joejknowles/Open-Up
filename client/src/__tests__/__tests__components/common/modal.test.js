import React from 'react';
import renderer from 'react-test-renderer';
import Modal from '../../../components/common/modal';

it('renders as before', () => {
  const tree = renderer.create(
    <Modal>
      modal children
    </Modal>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
