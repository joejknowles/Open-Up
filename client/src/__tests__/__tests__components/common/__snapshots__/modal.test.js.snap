exports[`test renders as before 1`] = `
<div
  className="modal"
  onClick={undefined}>
  <div
    className="modal-box"
    onClick={[Function]}>
    modal children
  </div>
</div>
`;
